docs/en/stack/ml/get-started-trial.asciidoc

```diff
@@ -8,13 +8,13 @@ image::images/management-license.png["The License Management page in {kib}"]
 For more information about Elastic license levels, see 
 https://www.elastic.co/subscriptions.
 
-You can start a 30-day trial to try out all of the platinum features, including 
-{ml-features}. Click **Start trial** on the **License Management** page in {kib}.
+You can start a 30-day trial to try out all of the subscription features,
+including {ml-features}. Click **Start trial** on the **License Management**
+page in {kib}.
 
-IMPORTANT: If your cluster has already activated a trial license for the current 
-major version, you cannot start a new trial. For example, if you have already 
+IMPORTANT: If your cluster has already activated a trial for the current major
+version, you cannot start a new trial. For example, if you have already 
 activated a trial for v7.0, you cannot start a new trial until v8.0.
```

Comment:
Could we make this paragraph consistent with others that are reporting the option to extend the trial?